fe/fe-core/src/main/java/com/starrocks/external/HiveMetaStoreTableUtils.java

```diff
@@ -210,6 +213,9 @@ public static Type convertColumnType(String hiveType) throws DdlException {
             case "BOOLEAN":
                 primitiveType = PrimitiveType.BOOLEAN;
                 break;
+            case "BINARY":
```

Comment:
Not only `Binary`. All unsupported types should be captured.

Reply:
> Not only `Binary`. All unsupported types should be captured.

```
            case "BINARY":
            case "MAP":
            case "STRUCT":
            case "UNIONTYPE":
                primitiveType = PrimitiveType.UNKNOWN_TYPE;
                break;
```
Now complex types and binary type will all be converted to UNKNOWN_TYPE.

Reply:
why not capture it in `default` branch?

Reply:
> 

done